pages/content/posts/2022-q3-report/index.md

```diff
@@ -0,0 +1,186 @@
+
+## The contribution of Open Web Docs
+
+Open Web Docs is the most significant organization contributing to MDN content maintenance. Of the multiple MDN organization repositories supported by Open Web Docs, the busiest is **mdn/content**, which contains the source for MDN’s pages. This repository is extremely active:
+
+* mdn/content is the 5th most active repository on all of GitHub, as measured by the number of commits/month according to [https://git-pulse.github.io/snapshots/](https://git-pulse.github.io/snapshots/).
+* In Q3 2022, 2,607 PRs were merged into mdn/content from 535 unique contributors.
+
+This report will demonstrate OWD’s contribution to the day-to-day maintenance of mdn/content with two metrics:
+
+* **Quarterly PRs merged to the mdn/content repository.**
+* **Quarterly number of reviews performed on each merged PR to the mdn/content repository.**
+
+Both these metrics contain contributions by OWD team members, other organizations that help maintain the repository, and volunteer contributors. Contribution metrics are divided into the following groups:
+
+* OWD staff
+* W3C staff (this is actually just Mike Smith)
+* Mozilla staff
+* Other (mostly volunteers, but also people paid by other organizations to work on MDN)
+
+As [@queengooborg](https://github.com/queengooborg) works for both OWD and Mozilla, her contributions have been evenly split between each organization.
+
+### PRs merged to mdn/content
+
+<table>
+  <tr valign="top">
+   <td><strong>PRs merged</strong>
+   </td>
+   <td><code>All merged PRs: 2607</code><br>
+       <code>OWD: 284.5</code><br>
+       <code>W3C: 42</code><br>
+       <code>Mozilla: 136.5</code><br>
+       <code>Other: 2144</code><br>
+       <img src="prs.png" width="750px" alt="Percentage pie chart. OWD: 10.9%, Mozilla: 5.2%, W3C: 1.6%, Other: 82.2%">
+   </td>
+  </tr>
+</table>
+
+This demonstrates how, of organizations contributing to mdn/content as measured by PR volume, OWD is the biggest contributor. It also shows that mdn/content gets a huge volume of contributions from volunteers: 82.2% of PRs in Q2 were “Other”, which is mostly volunteers.
+
+### Reviews of mdn/content PRs
+
+<table>
+  <tr valign="top">
+   <td><strong>Reviews on merged PRs</strong>
+   </td>
+   <td><code>All merged PRs: 3090</code><br>
+       <code>OWD: 1942</code><br>
+       <code>W3C: 349</code><br>
+       <code>Mozilla: 193</code><br>
+       <code>Other: 603</code><br>
+       <img src="reviews.png" width="750px" alt="Percentage pie chart. OWD: 62.9%, W3C: 11.3%, Mozilla: 6.2%, Other: 19.5%">
+   </td>
+  </tr>
+</table>
+
+The small OWD team provides the largest number of PR reviews to mdn/content, by a significant margin. OWD technical writers performed 63% of all reviews on merged PRs in the quarter.
+
+# Projects
+
+In the third quarter of 2022, OWD completed the projects listed below.
+
+## Modernization of JavaScript docs
```

Comment:
This is not just "JavaScript docs" but is "All JS code samples on MDN" isn't it?

I think this is a great section, the only comment really is that I don't get a sense of the scale of this change. It is possible to give an idea? How many pages, roughly, have been updated in this project? How many PRs? (I mean, it's impressive because it's a really huge project, not because updating examples to use modern JS is particularly hard).

I don't think this is essential but if it's not much work, maybe worth doing.

Reply:
"Modernizing code samples on MDN" might be the better heading here. What do you think, @teoli2003?

Reply:
It also says, just below:

> The MDN Web Docs JavaScript documentation dates back to ES3. The docs were completely revamped around 2015 (for ES6), however, since then the docs didn’t see a major update throughout. New features were documented and used latest best practices, but docs for older features mostly demonstrate best practices of the time.

...which strongly implies that the scope of this was the JS docs.

Reply:
I added a paragraph to link this historic to all code samples.